wolfssl/wolfcrypt/port/Renesas/renesas-tsip-crypt.h

```diff
@@ -202,15 +108,305 @@ typedef wolfssl_TSIP_Hash wc_Sha;
+        const byte* sig,    word32  sigSz,
+        const byte* hash,   word32  hashSz,
+        const byte* key,    word32  keySz,
+        int*  result, void*   ctx);
+
+WOLFSSL_LOCAL int wc_tsip_generateVerifyData(
+        const uint8_t*  masterSecret,
+        const uint8_t*  side,
+        const uint8_t*  handshake_hash,
+        uint8_t*        hashes);
+
+WOLFSSL_LOCAL int wc_tsip_AesCbcEncrypt(
+        struct Aes* aes,
+        byte*       out,
+        const byte* in,
+        word32      sz);
+
+WOLFSSL_LOCAL int wc_tsip_AesCbcDecrypt(
+        struct Aes* aes,
+        byte*       out,
+        const byte* in,
+        word32      sz);
+ 
+WOLFSSL_LOCAL int wc_tsip_AesGcmEncrypt(
+        struct Aes* aes, byte* out,
+        const byte* in, word32 sz,
+              byte* iv, word32 ivSz,
+              byte* authTag, word32 authTagSz,
+        const byte* authIn, word32 authInSz,
+        void* ctx);
+            
+WOLFSSL_LOCAL int wc_tsip_AesGcmDecrypt(
+        struct Aes* aes, byte* out,
+        const byte* in, word32 sz,
+        const byte* iv, word32 ivSz,
+        const byte* authTag, word32 authTagSz,
+        const byte* authIn, word32 authInSz,
+        void* ctx);
+
+WOLFSSL_LOCAL int wc_tsip_ShaXHmacVerify(
+        const struct WOLFSSL *ssl,
+        const byte* message, 
+        word32      messageSz,
+        word32      macSz,
+        word32      content);
+
+WOLFSSL_LOCAL int wc_tsip_Sha1HmacGenerate(
+        const struct WOLFSSL *ssl,
+        const byte* myInner, 
+        word32      innerSz,
+        const byte* in,
+        word32      sz, 
+        byte*       digest);
+
+WOLFSSL_LOCAL int wc_tsip_Sha256HmacGenerate(
+        const struct WOLFSSL *ssl,
+        const byte* myInner,
+        word32      innerSz,
+        const byte* in,
+        word32      sz,
+        byte*       digest);
+
+WOLFSSL_LOCAL int  tsip_Open();
+
+WOLFSSL_LOCAL void tsip_Close();
+
+WOLFSSL_LOCAL int  tsip_hw_lock();
+
+WOLFSSL_LOCAL void tsip_hw_unlock( void );
+
+WOLFSSL_LOCAL int  tsip_usable(const struct WOLFSSL *ssl,
+                                uint8_t session_key_generated);
+
+WOLFSSL_LOCAL void tsip_inform_sflash_signedcacert(
+        const byte* ps_flash, 
+        const byte* psigned_ca_cert,
+            word32  len);
+
+WOLFSSL_LOCAL byte tsip_rootCAverified();
+
+WOLFSSL_LOCAL byte tsip_checkCA(word32 cmIdx);
+
+WOLFSSL_LOCAL int  wc_tsip_tls_RootCertVerify(
+        const   byte* cert,   word32 cert_len,
+        word32  key_n_start,  word32 key_n_len,
+        word32  key_e_start,  word32 key_e_len,
+        word32  cm_row);
+
+WOLFSSL_LOCAL int  wc_tsip_tls_CertVerify(
+        const   uint8_t* cert,      uint32_t certSz,
+        const   uint8_t* signature, uint32_t sigSz,
+        uint32_t  key_n_start,      uint32_t key_n_len,
+        uint32_t  key_e_start,      uint32_t key_e_len,
+        uint8_t*  tsip_encRsaKeyIdx);
+/*----------
+WOLFSSL_LOCAL void tsip_inform_key_position(
```

Comment:
Looks like we should delete this commented out function prototype?

Reply:
Removed that commented portion.